android/src/main/java/com/instabug/flutter/modules/InstabugApi.java

```diff
@@ -225,6 +233,37 @@ public void clearAllExperiments() {
         Instabug.clearAllExperiments();
     }
 
+    @Override
+    public void addFeatureFlags(@NonNull Map<String, String> featureFlags) {
+        try {
+            List<IBGFeatureFlag> features = new ArrayList<>();
+            for (Map.Entry<String, String> entry : featureFlags.entrySet()) {
+                features.add(new IBGFeatureFlag(entry.getKey(), entry.getValue().isEmpty() ? null : entry.getValue()));
+            }
```

Comment:
BTW we can use Java 8 streams here instead if you think they will be more readable, I'm okay with keeping this one as well.